### ISSUE TITLE
MSMARCO Dataset for Listwise Loss Functions

### ISSUE THREAD

**Issue by shirinssalehi** [tensorflow/ranking#287]
Hello,

I have a question on how the MSMARCO dataset is used for the listwise loss function. In the MSMARCO dataset there is only one relevant document per query (in average) while the listwise loss function deals with a list of relevant documents. In addition, the original dataset does not provide any relevance score for the documents, and there is only a binary label of relevance or irrelevance. How should we prepare the dataset to be applicable to the listwise loss functions?
